api/src/main/java/com/epam/pipeline/manager/metadata/MetadataManager.java

```diff
@@ -407,7 +403,7 @@ private void checkEntityExistsAndCanBeModified(final Long entityId, final AclCla
     private Object loadEntity(final Long entityId, final AclClass entityClass) {
         Object entity;
         if (entityClass.equals(AclClass.ROLE)) {
```

Comment:
since now Role is a `SecuredEntity` we can support same way to load it as for all other types of `SecuredEntity`